src/pages/auth/Auth.context.tsx

```diff
@@ -15,11 +15,11 @@ import {
 const Context = createContext({});
 
 export interface AuthContextProps {
-  error: string | null;
-  user: User;
-  accounts: Array<ConnectedAccount>;
-  loginOauth: (p: Provider) => Promise<LoginOauthReturnValue>;
-  logout: () => void;
+  error?: string | null;
+  user?: User;
+  accounts?: Array<ConnectedAccount>;
+  loginOauth?: (p: Provider) => Promise<LoginOauthReturnValue>;
+  logout?: () => void;
```

Comment:
I guess there is no need to change the signature here, probably could be handled in a different way. I'll take a look.